mmengine/optim/scheduler/param_scheduler.py

```diff
@@ -667,21 +673,29 @@ def build_iter_from_epoch(cls,
 
     def _get_value(self):
         """Compute value using chainable form of the scheduler."""
+
+        def _get_eta_min(base_value):
+            if self.eta_min_ratio is None:
+                return self.eta_min
+            return base_value * self.eta_min_ratio
```

Comment:
Since `_get_eta_min` is only called with `base_value`. We can directly calculate the minimum value before the if else, rather than define a nested function